runtime/src/native_loader.rs

```diff
@@ -132,11 +132,16 @@ impl NativeLoader {
         let params = keyed_accounts_iter.as_slice();
         let name_vec = &program.try_account_ref()?.data;
         let name = match str::from_utf8(name_vec) {
-            Ok(v) => v,
+            Ok(v) => v.trim_end_matches(char::from(0)),
```

Comment:
wdyt about just checking `v.starts_with('\0')`?    I think that's about the smallest unit of work that'll tell us if the account is not initialized.

Reply:
Yeah, that's what I originally had, I can go back to that